show/main.py

```diff
@@ -1057,5 +1057,19 @@ def ecn():
     proc = subprocess.Popen(cmd, stdout=subprocess.PIPE, shell=True)
     click.echo(proc.stdout.read())
 
+#
+# 'reboot-cause' command ("show reboot-cause")
+#
+@cli.command('reboot-cause')
+def reboot_cause():
+    """Show cause of most recent reboot"""
+    PREVIOUS_REBOOT_CAUSE_FILE = "/var/cache/sonic/previous-reboot-cause.txt"
+    if not os.path.isfile(PREVIOUS_REBOOT_CAUSE_FILE):
```

Comment:
>isfile [](start = 19, length = 6)

So you mean after each unexpected reboot, the file will be missing? Could you explain the reason as code comment?

Reply:
Added comment.